tests/algorithms/test_bc.py

```diff
@@ -152,7 +151,12 @@ def test_smoke_bc_creation(
     expert_data_type=expert_data_types,
     rng=rngs,
 )
-@hypothesis.settings(deadline=20000, max_examples=15)
+@hypothesis.settings(
+    deadline=20000,
+    max_examples=15,
+    # TODO: one day consider removing this. For now we are good.
```

Comment:
maybe this is obvious for someone who's more familiar with `hypothesis` or this codebase but what does this check do? would it be helpful to clarify when it would be possible to remove this check or what's going that causes this check to fail?

Reply:
Thanks that is a good point! I added a better description. Does it make sense now @tomtseng ?